docs/griptape-cloud/tools/run-tool.md

```diff
@@ -0,0 +1,26 @@
+# Running a Tool
+
+Once your Tool is created and deployed, you can run your Tool one of three ways outlined below. You may view the details of any of your runs, no matter how you created them, in the `Runs` tab of your Tool.
```

Comment:
> no matter how you created them,

Feels unneeded.

Reply:
i copied that wording from the structures tab